src/components-examples/material/paginator/BUILD.bazel

```diff
@@ -42,8 +42,8 @@ ng_test_library(
         ":paginator",
         "//src/cdk/testing",
         "//src/cdk/testing/testbed",
-        "//src/material/paginator",
-        "//src/material/paginator/testing",
+        "//src/material/legacy-paginator",
```

Comment:
need to invert the dep

Reply:
I thought with examples we were leaving them as-is?

Reply:
Ah yep, too many build files in this PR and I got confused which one I was looking at. You're good